### ISSUE TITLE
Mod conflictiong with Arumba's Angel + Bob's Mod Pack

### ISSUE THREAD

**Issue by soncemvo** [Yousei9/Factorio-LogisticTrainNetwork#6]
Where you try to install LogisticTrainNetwork with Arumba's Angel + Bob's Mod Pack you got error:
![image](https://cloud.githubusercontent.com/assets/862204/22426211/55a118d8-e720-11e6-988b-84f6d4a9053d.png)

**Comment by Yousei9** [comment]
I get similar errors with that mod pack for other mods than LTN.
Ask the mod pack maintainer for help.

**Comment by Cube2M16** [comment]
No, this is a problem with your mod. I looked, and there was no problem before I installed your mod. There is no such path called ArumbaAngelBobs/color-coding/graphics/icons/rail-tanker.png. There is a ArumbaAngelBobs/color-coding/graphics, but there is no icons folder inside the graphics folder. Furthermore, there is nothing that has to do with rail tanker in there. On the other hand, there is a LogisticTrainsNetwork/graphics/icons/rail-tanker.png. Combine this with how the error didn't happen until your mod was installed, your mod is the cause of this error. Now if you would like more details on the issue, or help debugging, I would be happy to help, but if not then i am not downloading LTN. Shame, I thought it looked fun, but I'll just make do with another, more crappy train mod.

**Comment by Cube2M16** [comment]
I apologize for my last comment. I was just angry and then looking for more information I went to the forum for ur mod. I looked at the changelog, and the. For some reason, I clicked the link to another forum post which happened to be talkig about this problem. I saw ur post saying you would fix it, and realized shit I was just really rude to someone who fixed it after all. Anyway I looked at the time you posted that and it said March 28 2017 5:30 pm or something, so while technically I posted this comment before you replied in the forum,(it's 4:02 pm where I live) I don't think the loophole would be appreciated. Thank you for fixing this after all, and I suggest you mention that this is fixed and remove the wontfix tag from it. I trust the update released 2 hours ago contains this fix? Thank you!
